.github/workflows/rp-docker-build-push-ecr-dev.yml

```diff
@@ -7,6 +7,7 @@ on:
     paths:
       - "simple-rp-simulator/**"
       - ".github/workflows/rp-docker-build-push-ecr-dev.yml"
+  workflow_dispatch:
```

Comment:
This allows us to manually trigger the workflow. It's nice to have this available for debugging purposes in our lower environments.